src/System.Runtime.InteropServices.RuntimeInformation/src/System.Runtime.InteropServices.RuntimeInformation.csproj

```diff
@@ -12,6 +12,9 @@
     <PackageTargetFramework>dotnet5.1</PackageTargetFramework>
     <PackageTargetRuntime Condition="'$(TargetsWindows)' == 'true'">win</PackageTargetRuntime>
     <PackageTargetRuntime Condition="'$(TargetsUnix)' == 'true'">unix</PackageTargetRuntime>
+    <DefineConstants Condition="'$(TargetGroup)'=='net46'">net46</DefineConstants>
```

Comment:
It would be preferred to avoid these defines and instead factor out the code into partial classes into different cs files.